docs/rfc/rfc-5.md

```diff
@@ -0,0 +1,271 @@
+* Map the Mezo `token` address to the proper token address on the target `chain`:
+    * If `token` is BTC, the target token should be TBTC on Ethereum, regardless of the value of the `chain`
+      parameter. TBTC address is returned by the `getSourceBTCToken` method of the `AssetsBridge` precompile
+    * Otherwise, the target token should be determined using bridge mappings returned by the
+      `getERC20TokensMappings` method of the `AssetsBridge` precompile
+* Create a new `AssetsUnlocked` storage entry in the `x/bridge` module. `AssetsUnlocked` should contain all
+  input parameters of the given `bridgeOut` call and additionally:
+    * Have an `unlockSequence` field allowing to uniquely identify the given `AssetsUnlocked` entry
+    * Modify the `token` field to represent the token address on the **target** chain, as resolved from the mapping
+* Emit appropriate EVM events
+
+Changes described in this section should be covered with a comprehensive
+[system test](https://github.com/mezo-org/mezod/tree/main/tests/system) suite.
+
+### Ethereum sidecar
+
+Each Mezo validator runs an instance of the Ethereum sidecar that feeds the existing 1-way bridge with events
+emitted on the Ethereum chain. This proposal assumes re-using this infrastructure for bridging out of Mezo.
+
+The Ethereum sidecar should be enhanced with an ability to fetch recent `AssetsUnlocked`entries submitted on the
+Mezo chain and attest them on Ethereum. The specific logic described below is supposed to be executed only by
+validators having the "bridge" privilege (the so called bridge validators). List of those validators can be
+determined by calling the `validatorsByPrivilege` method (with `1` as parameter) exposed by the `ValidatorPool`
+precompile.
+
+#### Monitoring loop
+
+The Ethereum sidecar should query its validator node privately to fetch recent `AssetsUnlocked` entries stored in
+the `x/bridge` module. The query mechanism is an implementation detail and is not enforced by this document. A
+possible approach is exposing `AssetsUnlocked` entries by the validator using EVM JSON-RPC or gRPC and periodically
+polling them from within the sidecar monitoring loop. What is important, the sidecar should not query any third-party
+node to avoid centralization and putting trust on external components.
+
+#### Attestation process
+
+Once `AssetsUnlocked`entries are fetched by the sidecar, all of them should be attested on Ethereum. The
+attestation process requires access to the operator private key used by the corresponding validator. This is necessary
+to attribute the given attestation to a specific bridge validator (as returned by the aforementioned
+`validatorsByPrivilege` method of the `ValidatorPool` precompile). The integration of the private key is an
+implementation detail but a possible solution may be using the keyring from within the sidecar process.
```

Comment:
If we reuse this key, let's make sure we CAPSLOCK in the code to not reuse it anywhere else. Single-time operator registration and the bridge-out attestations are fine but adding anything else in betwee slightly increases the risk.